taccsite_cms/templates/nav_search.raw.html

```diff
@@ -5,28 +5,27 @@
 <style type="text/css">
   /* Hide search bar before all relevant styles have loaded */
   /* FAQ: Manage visibility of search bar to avoid FOUC */
-  /* HACK: This should be done via a dynamically-added `-is-loaded` class */
+  /* HACK: GH-101: Resolve via dynamically-added `[data-status="loaded"]` */
   :host { visibility: hidden; height: 0; }
 </style>
-
-<link id="css-site" rel="stylesheet" href="{% static 'site_cms/css/build/site.css' %}">
-
 <link rel="stylesheet" href="{% static 'site_cms/css/build/site.tacc-search-bar.css' %}">
-<link rel="stylesheet" href="{% static 'site_cms/css/build/freeze_variables/site.tacc-search-bar.css' %}">
```

Comment:
**Explanation.** This file/markup is dynamically added into custom element. Some of the styles it needed are in `site.css`. But this PR moves all (and only) the styles it needs into `site.tacc-search-bar.css`.